VTEX - Logistics API.json

```diff
@@ -6996,279 +6988,166 @@
                             "application/json": {
                                 "schema": {
                                     "type": "array",
+                                    "description": "List of pickup points, limited to 1.000 pickup points.",
```

Comment:
I think the description should mention the endpoint that returns a paged list as an alternative if you need more than 1000 results.